v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -397,6 +456,62 @@ When passing a chained role into `BACKUP`, it will follow this pattern with each
 BACKUP DATABASE <database> INTO 'gs://{bucket name}/{path}?AUTH=implicit&ASSUME_ROLE={intermediate service account name}iam.gserviceaccount.com,{final service account name}iam.gserviceaccount.com'; AS OF SYSTEM TIME '-10s';
 ~~~
 
+### Google Cloud workload identity 
+
+With a CockroachDB cluster deployed on [Kubernetes](kubernetes-overview.html), you can allow your pods to authenticate as an IAM service account that you have associated to a Kubernetes service account. You can then use assume role authentication to allow the IAM service account to assume another service account that has permissions to perform bulk operations to a Google Cloud Storage bucket.
+
+This means that a CockroachDB node will only be able to access credentials for the IAM service account associated with the Kubernetes service account.
+
+You can use workload identities with assume role authentication to run the following operations:
+
+- [`BACKUP`](backup.html)
+- [`CREATE CHANGEFEED`](create-changefeed.html)
+- [`EXPORT`](export.html)
+- [`IMPORT`](import.html)/[`IMPORT INTO`](import-into.html)
+- [`RESTORE`](restore.html)
+
+{{site.data.alerts.callout_info}}
+Service accounts in Google and Kubernetes refer to different resources. See [Google's documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity#terminology) for definitions.
+{{site.data.alerts.end}}
+
+Since there are multiple IAM service accounts referred to in this section, see the following descriptions:
+
+- _Identity_: the IAM service account you have associated with your Kubernetes service account.
```

Comment:
Should we use "Identity Service Account" and "Operation Service Account" to make it clear that they are service accounts? (or "Identity SA" can be fine too).